APMrover2/mode.cpp

```diff
@@ -205,7 +205,7 @@ void Mode::set_desired_location(const struct Location& destination, float next_l
     // set final desired speed
     _desired_speed_final = 0.0f;
     if (!is_equal(next_leg_bearing_cd, MODE_NEXT_HEADING_UNKNOWN)) {
-        const float curr_leg_bearing_cd = get_bearing_cd(_origin, _destination);
```

Comment:
I'd like to clarify this name.  Is it bearing to or from?

`get_bearing_to`, perhaps?